angular-velocity.js

```diff
@@ -252,24 +250,29 @@
 	}
 
 	// Use the factories to define animations for all velocity's sequences
-	angular.forEach(Velocity.RegisterEffect.packagedEffects, function (_, animation) {
-		var IN = 'In',
-			OUT = 'Out',
-			selector = '.' + animationToClassName(animation),
-			oppositesSelector = '.' + animationToOppositesClassName(animation),
-			enterSelector, leaveSelector;
-
-		app.animation(selector, makeAngularAnimationFor(animation));
-
-		if (animation.substr(-2) === IN) {
-			enterSelector = '.' + animationToEnterClassName(animation);
-			app.animation(oppositesSelector, makeOppositesAnimationFor(animation));
-			app.animation(enterSelector, makeEnterAnimationFor(animation));
-		}
-		else if (animation.substr(-3) === OUT) {
-			leaveSelector = '.' + animationToLeaveClassName(animation);
-			app.animation(leaveSelector, makeLeaveAnimationFor(animation));
-		}
-	});
+	angular
+		.module('angular-velocity', ['ngAnimate'])
+		.config(['$animateProvider', function ($animateProvider) {
+			// Use the factories to define animations for all velocity's sequences
+			angular.forEach(Velocity.Redirects, function (_, animation) {
+				var IN = 'In',
+					OUT = 'Out',
```

Comment:
Could you fix the indentation before merge? I can do it before minification if you'd rather that, just let me know.